rubygem-azure-core/rubygem-azure-core.spec

```diff
@@ -0,0 +1,89 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+# Generated from azure-0.7.5.gem by gem2rpm -*- rpm-spec -*-
+%global gem_name azure
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.7.5
+Release: 1%{?dist}
+Summary: Official ruby client library to consume Microsoft Azure services
```

Comment:
Please swap this comment with https://github.com/theforeman/foreman-packaging/pull/1140#discussion_r66590923!